etlman/templates/projects/new_step.html

```diff
@@ -1,22 +1,36 @@
 {% extends "base.html" %}
 {% load crispy_forms_tags %}
 {% block content %}
+<!-- Remove me before merging -->
```

Comment:
@tobiasmcnulty for some reason HTMX was not working on this page until I imported it "again" - should be in `base.hmtl`. Any idea why this could be?

Reply:
Merging for now. We will revisit this issue.